### ISSUE TITLE
As an Admin, I want to have clear instructions for switching to Dev Workspace engine

### ISSUE THREAD

**Issue by ibuziuk** [eclipse/che#20614]
### Is your task related to a problem? Please describe

As an Admin, I want to have clear instructions for swithing from Che Server to Dev Workspace Engine. 
![image](https://user-images.githubusercontent.com/1461122/136569060-2b3f1ab0-c77e-4bf0-b1a9-8dea515b1166.png)

### Describe the solution you'd like

As an Admin, I want to have clear instructions for swithing from Che Server to Dev Workspace Engine (stable or next channels).

**Comment by ibuziuk** [comment]
SOP that I tried so far, and which does not work as expected (details below):

- install 'eclipse-che' using the stable channel
- create CR with a single-host and OpenShift Auth enabled 
- create an account (using 'developer' OpenShift account on the crc for example) with a couple of workspaces and start one of them
- wait until workspace is running, run build / run command
- start preparations for the migration procedure from the `stable` to `tech-preview-stable-all-namespace` 
- delete `eclipse-che` subscription from the namespace
```
[ibuziuk@fedora ~]$ oc get subscription
NAME          PACKAGE       SOURCE                CHANNEL
eclipse-che   eclipse-che   community-operators   stable
[ibuziuk@fedora ~]$ oc delete subscription eclipse-che
subscription.operators.coreos.com "eclipse-che" deleted
```
- delete csv and dependent objects
![image](https://user-images.githubusercontent.com/1461122/136939650-68264761-b18c-4bd1-adf5-d4d948fe15aa.png)
- operator pod in the namespace will be terminated
![image](https://user-images.githubusercontent.com/1461122/136939785-4ce7ca52-25c5-4f90-834a-bfcd6c3e956c.png)
- install 'eclipse-che' using the `tech-preview-stable-all-namespace`
![image](https://user-images.githubusercontent.com/1461122/136939892-aafdd299-a146-473a-9c47-5bf71022cdeb.png)
- operators are installed in the `all-namespace` mode as expected
![image](https://user-images.githubusercontent.com/1461122/136940264-78ac2212-3e4f-4d9a-b6ee-d1778e5075d9.png)
- update the CR from the `eclipse-che` namespace and enable devWorkspace
```
  devWorkspace:
    enable: true
```
- go back to the running workspace
- session expired / re-login is required
- login using the 'developer' account that was used previously
- ERROR: on the 'eclipse-che' end the new account is created 

![image](https://user-images.githubusercontent.com/1461122/136941334-226aae5d-b495-4cad-9c7f-21b0064b2804.png)

- ERROR: UD shows the following errors
![image](https://user-images.githubusercontent.com/1461122/136941444-f332e3ab-371e-4e82-a75e-7e4a567dba67.png)

- Start a new DevWorkspace
- ERROR: Workspace is started, but since the account is different, in a different namespace (not `developer-dev`, but 
`developeryv6z-dev`)
![image](https://user-images.githubusercontent.com/1461122/136941647-d981a7fb-0801-4e47-8ec9-37f2d830ce2a.png)
- ERROR: previously running workspace pod (the one that has been started before migration in the `developer-che` namespace) is still running.

Basically, there are two main issues with the switch from the 'stable' to `tech-preview-stable-all-namespace` channel
1. for some reason, the new account is created after the devWorkspace switch

dbche=# SELECT * from account;
                  id                  |     name      |   type   
--------------------------------------+---------------+----------
 434d4bdf-7bae-47de-b657-9c3d675bf4d2 | developer     | personal
 e8a1a712-743c-4978-b095-9b3db488a110 | developeryv6z | personal
(2 rows)

2. previously running workspaces managed by che-server are not stopped (when admin switches the engine to DevWorskpace all che-server based workspaces should be stopped)

**Comment by benoitf** [comment]
moving issue to 7.42 milestone as this issue is part of the endgame

**Comment by ibuziuk** [comment]
Fixed as part of https://github.com/eclipse-che/che-docs/pull/2269
